### PR TITLE
GitHub action to publish Java/Scala

### PR DESCRIPTION
**Pull request by richard-rogers**
### General Checklist

* [ ] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [ ] Conform by the style guides, by using formatter
* [ ] Documentation updated
* [ ] (optional) Please add a label to your PR

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: whylogs CI
+ 11name: whylogs CI
```
Comment (jamie256) [inline .github/workflows/continuous-integration.yml]: Is this intentional?

Comment (richard-rogers) [reply .github/workflows/continuous-integration.yml]: Still getting used to the new keyboard :)

**Review by jamie256 — APPROVED** [review]
Looks good!

File: .github/workflows/java-continuous-integration.yml [hunk 2]
```diff
@@ -1,7 +1,9 @@
-name: Run Gradle on PRs
+name: whylogs Java CI
```
Comment (jamie256) [inline .github/workflows/java-continuous-integration.yml:1]: Better name, thanks!

File: .github/workflows/java-continuous-integration.yml [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
 on:
+  push:
+    branches: [ mainline, release]
   pull_request:
-    branches: [ mainline ]
+    branches: [ mainline, release]
```
Comment (jamie256) [inline .github/workflows/java-continuous-integration.yml:6]: Great, so now we are consistent with the triggers on the python changes?

**Review by jamie256 — APPROVED** [review]
Nice clean up and integrating the java CI better!

**Pull request merged**